packages/components/src/internal/renderers/StoredAmountRenderer.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { List, Map } from 'immutable';
+
+import { ValueDescriptor } from '../components/editable/models';
+
+import { DefaultRenderer } from './DefaultRenderer';
+
+interface Props {
+    data: Map<any, any>;
+}
+
+export class StoredAmountRenderer extends React.PureComponent<Props, any> {
+    static getEditableRawValue = (values: List<ValueDescriptor>): string[] => {
+        return values.size === 1 ? values.first()?.display : undefined;
+    };
+    static getOriginalRawValue = (values: any): string[] => {
```

Comment:
These methods are both returning the display value, which doesn't usually mean "raw value".  I know it's because this column is different, but it still makes my brain hurt a little. Would `getEditableValue` and `getOriginalEditableValue` work as names?

Reply:
getEditableRawValue was an existing method name used by the AliasRenderer. I'll leave it as is for 24.2, but we can adjust in develop.